tree/dataframe/src/RDFHelpers.cxx

```diff
@@ -24,6 +24,41 @@
 #include <algorithm>
 #include <set>
 
+#include <numeric>
+#ifndef _WIN32
+#include <unistd.h>
+#endif
+#include <stdio.h>
+
+#include <unistd.h>
+#include <iostream>
+
+#if defined(_WIN32)
+#define WIN32_LEAN_AND_MEAN
+#define VC_EXTRALEAN
+#include <Windows.h>
+#elif defined(__linux__)
+#include <sys/ioctl.h>
+#endif // Windows/Linux
+
+// Get terminal size for progress bar
+int get_tty_size() {
+#if defined(_WIN32)
+    int width = 0;
+    CONSOLE_SCREEN_BUFFER_INFO csbi;
+    GetConsoleScreenBufferInfo(GetStdHandle(STD_OUTPUT_HANDLE), &csbi);
+    width = (int)(csbi.srWindow.Right-csbi.srWindow.Left+1);
+    return width;
+#elif defined(__linux__)
+    int width = 0;
+    struct winsize w;
+    ioctl(fileno(stdout), TIOCGWINSZ, &w);
+    width = (int)(w.ws_col);
+    return width;
+#endif // Windows/Linux
```

Comment:
Not strictly related to this PR, but probably this kind of code should belong to common, core libraries instead?